syft/frameworks/keras/model/sequential.py

```diff
@@ -58,7 +58,6 @@ def share(model, *workers, target_graph=None):  # pragma: no cover
 
     # Push and initialize shared model on servers
     sess = tfe.Session(graph=target_graph)
-    tf.Session.reset(sess.target)
```

Comment:
Should we add coverage back to this function?

Reply:
What do you mean here?

Reply:
at the top of the function there is a comment of "#pragma: no cover" which prevents coverage from being run on this function. We should remove that if this function is now being tested

Reply:
Gotcha, and done!

Reply:
Did you push your changes?

Reply:
jup